crmsh/bootstrap.py

```diff
@@ -1164,26 +1241,32 @@ def gen_mcastaddr():
     csync2_update(corosync.conf())
 
 
+def profiles_adjust_for_corosync():
```

Comment:
I don't understand this method. Why do we call it `adjust_for_corosync` and we only check the `totem` entry?
What happens with everything else.
If we only want to modify the totem, maybe we should rename the method `to adjust_corosync_totem`

Reply:
Changed as a more flexible resolution